src/liquid/place/create_form.html

```diff
@@ -31,17 +31,15 @@
   </section>
 
   {% if contactType.user_role.size > 1 %}
-  <section class="section is-small">
+  <section class="section-box">
     {%
       include "components/contact_type_property.html" prefix="user_" prop=userRoleProperty
     %}
   </section>
   {% endif %}
 
-  <div class=" field is-grouped is-grouped-right">
-    <div class="control">
-      <button id="place_create_submit" class="button is-link">Save</button>
-      <a href="/" class="button">Cancel</a>
-    </div>
+  <div class="">
+    <button id="place_create_submit" class="btn">Save</button>
+    <a href="/" class="clear-btn" class="button">Cancel</a>
```

Comment:
Do we have both a `btn` and `button` class? Also the `Save` button casts a shadow but the cancel one doesn't even though they should be visually on the save level